src/db.js

```diff
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '123123',
```

Comment:
Consider removing the default password '123123' for security reasons. It's better to ensure that the password is always provided through environment variables.